relay-server/src/services/processor/attachment.rs

```diff
@@ -61,20 +62,21 @@ pub fn create_placeholders(
 pub fn scrub<Group>(managed_envelope: &mut TypedEnvelope<Group>, project_info: Arc<ProjectInfo>) {
     let envelope = managed_envelope.envelope_mut();
     if let Some(ref config) = project_info.config.pii_config {
-        let minidump = envelope
-            .get_item_by_mut(|item| item.attachment_type() == Some(&AttachmentType::Minidump));
-
-        if let Some(item) = minidump {
-            scrub_minidump(item, config);
-        } else if has_simple_attachment_selector(config) {
-            // We temporarily only scrub attachments to projects that have at least one simple attachment rule,
-            // such as `$attachments.'foo.txt'`.
-            // After we have assessed the impact on performance we can relax this condition.
-            for item in envelope
-                .items_mut()
-                .filter(|item| item.ty() == &ItemType::Attachment)
+        for item in envelope.items_mut() {
+            if project_info
+                .config
+                .features
+                .has(Feature::ViewHierarchyScrubbing)
```

Comment:
nit, maybe store the boolean in a temporary outside the loop, would make it a bit more readable.